resdk/resources/collection.py

```diff
@@ -64,6 +68,16 @@ def data(self, value):
         self._data = value
         self._data_hydrated = False
 
+    def add_data(self, *data):
```

Comment:
I don't know that is a bit hackish too. Let's sleep on this and plan it next year.